data/test/index.js

```diff
@@ -66,101 +68,112 @@ describe( 'select', () => {
 } );
 
 describe( 'withSelect', () => {
-	it( 'passes the relevant data to the component', () => {
-		registerReducer( 'reactReducer', () => ( { reactKey: 'reactState' } ) );
-		registerSelectors( 'reactReducer', {
-			reactSelector: ( state, key ) => state[ key ],
+	function cases( withSelectImpl, extraAssertions = noop ) {
```

Comment:
None of the tests themselves have changed, just putting in a function to allow testing both `withSelect` and `query` separately with same expectations (except `console.warn` in the latter).

Reply:
The function name is unclear to me :)

Reply:
>The function name is unclear to me :)

I broke my own rule of no abbreviations 😄 It's a semi-common convention for "Implementation". I've unabbreviated it.